providers/google/src/airflow/providers/google/cloud/operators/cloud_base.py

```diff
@@ -21,9 +21,14 @@
 
 from typing import Any
 
-from google.api_core.gapic_v1.method import DEFAULT
+from airflow.providers.google.version_compat import AIRFLOW_V_3_0_PLUS
```

Comment:
should not we mive the logic of import to `verson_compat.py` ?

Reply:
```python
from airflow.providers.google.version_compat import BaseOperator
```

Reply:
And in version_compat.py:

```python
if AIRFLOW_V_3_0_PLUS:
    from airflow.sdk import BaseOperator
else:
    from airflow.models import BaseOperator
```

Reply:
That would be way smaller and it's really what version_compat.py is for  :D

Reply:
We also discussed with @amoghrajesh the way of importing the _PLUS variable - that might be another "pattern" we coul enforce - just import what you need from "version_compat.py" and let it worry about compatibility :)

Reply:
Yup, fixed in https://github.com/apache/airflow/pull/52366/commits/5af9c118e72c8171a4d8d5e3f25cd07885efc416